src/components/login-form.js

```diff
@@ -1,14 +1,118 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import styled from "styled-components";
+import axios from "axios";
 
-export default function LoginForm() {
-  return <Wrapper>안녕 나는 로그인 폼!</Wrapper>;
-}
+export default function LoginForm({ isLoggedIn, setLoginStatus, loginCheck }) {
+  const [form, setForm] = useState({ email: "", password: "" });
+
+  const handleFormChange = (e) => {
+    setForm({
+      ...form,
+      [e.target.name]: e.target.value,
+    });
+  };
+  const initForm = () => {
+    setForm({ email: "", password: "" });
+  };
+  const validCheck = () => {
+    if (form.email.length === 0 || form.password.length === 0) {
+      alert("모든 항목을 입력하세요");
+      return false;
+    }
+    return true;
+  };
+  const handleSubmit = () => {
+    console.log(form);
+    if (!validCheck) return;
+    axios
+      .post(process.env.API_HOST + "/auth/signin/", form)
+      .then(function (response) {
+        console.log(response);
+        alert("로그인 성공!");
+        setLoginStatus(true);
+        console.log(isLoggedIn);
+      })
+      .catch(function (error) {
+        initForm();
+        alert("로그인 실패!");
+        console.log(error);
+      });
+  };
 
+  return (
+    <Wrapper>
+      <LoginLabel>로그인</LoginLabel>
+      <InfoInputArea>
+        <EmailInputArea>
+          <EmailLabel>EMAIL</EmailLabel>
+          <EmailInput
+            name="email"
+            value={form.email}
+            onChange={handleFormChange}
+          />
+        </EmailInputArea>
+        <PasswordInputArea>
+          <PasswordLabel>PASSWORD</PasswordLabel>
+          <PasswordInput
+            name="password"
+            value={form.password}
+            onChange={handleFormChange}
+            type="password"
+          />
+        </PasswordInputArea>
+      </InfoInputArea>
+      <Submit onClick={handleSubmit}>로그인</Submit>
+    </Wrapper>
```

Comment:
```suggestion
    <Wrapper>
      <Title>로그인</Title>
      <InputWrapper>
        <Row>
          <Label>EMAIL</Label>
          <Input
            name="email"
            type="text"
            value={form.email}
            onChange={handleFormChange}
          />
        </Row>
        <Row>
          <Label>PASSWORD</Label>
          <Input
            name="password"
            type="password"
            value={form.password}
            onChange={handleFormChange}
          />
        </Row>
      </InputWrapper>
      <SubmitButton onClick={handleSubmit}>로그인</SubmitButton>
    </Wrapper>
```
1. 컴포넌트 네이밍을 위와 같이 해보는 건 어떨까요? 컴포넌트들의 역할을 더 잘 파악할 수 있게욥
2. ```<EmailInputArea/>, <PasswordInputArea/>```, ```<EmailLabel/>, <PasswordLabel/>```, ```<EmailInput/>, <PasswordInput/>```은 style이 같기 때문에 굳이 컴포넌트를 따로 정의할 필요가 없어 보여요.
```Label, Input```같은 경우 Label 안과 Input name을 통해 어떤 것에 대한 label과 input을 파악할 수 있어 컴포넌트 네이밍에 굳이 또 명시할 필요는 없어 보입니당.
2. Input에는 type을 써주시는 게 좋아요 ㅎ
3. 핸들러 네이밍 좋네요 !